src/api/auth/auth.py

```
@@ -68,4 +72,119 @@ def token(
             headers={"WWW-Authenticate": "Bearer"},
         )
 
-    return issue_token(user)
\ No newline at end of file
+    return issue_token(user)
+
+
+@router.get("/google")
+def google_oauth(request: Request, code: str | None = None, state: str | None = None, db: Session = Depends(get_session)):
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)